katas/content/multi_qubit_systems/bell_state_change_1/Verification.qs

```diff
@@ -0,0 +1,13 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Katas;
+
+    operation BellStateChange1_Reference(qs : Qubit[]) : Unit is Adj + Ctl {
+        X(qs[0]);
+    }
+
+    @EntryPoint()
+    operation CheckSolution() : Bool {
+        CheckOperationsEquivalenceOnZeroStateWithFeedback(Kata.BellStateChange1, 
```

Comment:
Here we need to check that the solutions are equivalent once they start with a Bell state, not with the zero state. The easiest way to do this is define a wrapper operation that prepares the Bell state used as input and then applies to it `Kata.BellStateChange1`, and use `CheckOperationsEquivalenceOnZeroStateStrict` to compare the effect of that wrapper with an operation that prepares the target Bell state. If the results are different, you can use `ShowQuantumStateComparison` to show the difference, using Bell state prep as `initialState` argument.

I don't think we have an example of such approach in the modern Katas, only in the classic BasicGates. Ping me if anything is unclear!